### DISCUSSION TITLE
Dropwizard 2.0.24 compatibility with Oracle RAC

### DISCUSSION THREAD

**Discussion by altdelnow** [dropwizard/dropwizard#7241] [General]
Currently we are using dropwizard 1.0.6 in our  microservice application. We had upgraded dropwizard to 2.0.24 in one of the other application versions.

We want to check whether these dropwizard versions are compatible with Oracle RAC database. If one database node goes down, can dropwizard take care of re-establishing the connection objects without the need of restarting microservices?

**Comment by joschi** [comment]
Dropwizard 2.0.x is end-of-life since January 31, 2023: https://github.com/dropwizard/dropwizard/discussions/6324

The latest stable versions are Dropwizard 3.0.0 and 4.0.0.

Dropwizard is using jdbi3 or Hibernate ORM, together with the Tomcat JDBC Connection Pool which in turn use JDBC.

See https://www.dropwizard.io/en/stable/manual/jdbi3.html and https://www.dropwizard.io/en/stable/manual/hibernate.html for the respective documentation.
